dask/dataframe/groupby.py

```diff
@@ -68,6 +68,36 @@ def _max(g):
 def _count(g):
     return g.count()
 
+def _var_chunk(df, index):
+    if isinstance(df, pd.Series):
+        df = df.to_frame()
+    x = df.groupby(index).sum()
+    x2 = (df**2).rename(columns=lambda c: c + '-x2')
+    x2 = (pd.concat([df, x2], axis=1)
+            .groupby(index)
+            [[c + '-x2' for c in x.columns]]
+            .sum())
+    n = (df.groupby(index).count()
+           .rename(columns=lambda c: c + '-count'))
+
+    result = pd.concat([x, x2, n], axis=1)
+    return result
+
+def _var_agg(g, ddof):
+    g = g.groupby(level=0).sum()
+    nc = len(g.columns)
+    x = g[g.columns[:nc//3]]
+    x2 = g[g.columns[nc//3:2*nc//3]].rename(columns=lambda c: c[:-3])
+    n = g[g.columns[-nc//3:]].rename(columns=lambda c: c[:-6])
+
+    result = x2 - x**2 / n
```

Comment:
Generally we need to compute the sum of the squares of each numeric column and then group by the index.  We compute the variance through `E(x**2) - E(x)**2`.  This isn't the best approach, but it is the easiest to implement :)